Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -338,6 +339,13 @@ void ProjectData::parseProcesses(BaseLib::ConfigTree const& processes_config,
                         "dimension");
             }
         }
+        else if (type == "DensityDrivenFlow")
```

Comment:
@TomFischer said, that theoretically, you could run an HTC implementation as a purely HT setup, but this would not be as high-performing as an HT implementation. So he suggested to have several implementation. This would also fit into our development schedule, ie to build upon successive work any next step of complexity (eg HT -> HC -> HTC -> H_unsatTC).